### ISSUE TITLE
Страница города

### ISSUE THREAD

**Issue by nicothin** [nicothin/nesia#28]
@vladimir075 

Макеты: [настольная](https://www.figma.com/file/OMi3vX6CvJMMciuJ7rIvSI/Terra?node-id=3100%3A1883) и [мобильная](https://www.figma.com/file/OMi3vX6CvJMMciuJ7rIvSI/Terra?node-id=3172%3A2896)

Задача: сверстать.

**Comment by vladimir075** [comment]
верно. в мобильной версии запросил у дизайнера сделать некоторые исправления, потому начать верстать с настольной

**Comment by vladimir075** [comment]
под каруселями Property to buy и Property to rent хорошо посередине поставить надпись See all properties красным по примеру красной надписи See local highlights. Пропустили это в дизайне.

**Comment by vladimir075** [comment]
Еще если не сложно я бы попросил сделать вторую версию на пробу с переносом текста About Apokorona (заголовок, текст) на место между каруселью New properties for rent и блоком Neighbouring cities. Первую версию пока не стирать

**Comment by nicothin** [comment]
> > под каруселями Property to buy и Property to rent хорошо посередине поставить надпись See all properties красным по примеру красной надписи See local highlights. Пропустили это в дизайне.
> 
> Пропустили эту просьбу.

Извиняюсь.

У нас 3 страницы с похожим сочетанием блоков:

- https://nicothin.pro/nesia/index.html
- https://nicothin.pro/nesia/city.html
- https://nicothin.pro/nesia/landing.html

Добавил на все три, но для https://nicothin.pro/nesia/index.html попробовал добавить не текст, а кнопку. Взгляните, нужен ли такой акцент.



> Еще если не сложно я бы попросил сделать вторую версию на пробу с переносом текста About Apokorona (заголовок, текст) на место между каруселью New properties for rent и блоком Neighbouring cities. Первую версию пока не стирать

https://nicothin.pro/nesia/city.html - оригинал, https://nicothin.pro/nesia/city-2.html - со сменой очередности блоков.

**Comment by vladimir075** [comment]
интересный акцент, потом выберем.
Принято.
